opensrp-core/src/main/java/org/smartregister/service/FormSubmissionService.java

```diff
@@ -56,9 +56,7 @@ public void processSubmissions(List<FormSubmission> formSubmissions) {
                     updateFTSsearch(submission);
 
                 } catch (Exception e) {
-                    logError(format("Form submission processing failed, with instanceId: {0}. "
-                                    + "Exception: {1}, StackTrace: {2}", submission.instanceId(),
-                            e.getMessage(), ExceptionUtils.getStackTrace(e)));
+                    Timber.e(e, format("Form submission processing failed, with instanceId: %s", submission.instanceId()));
```

Comment:
Timber will automatically format so no need of calling the `format` method explicitly

Reply:
I had missed that. I've updated